site/docs/get-started/quickstart.md

```diff
@@ -8,7 +8,7 @@ Follow this guide to install Fybrik using default parameters that are suitable f
 
 Ensure that you have the following:
 
-- [Helm](https://helm.sh/) 3.3 or newer must be installed and configured on your machine.
+- [Helm](https://helm.sh/) 3.3 or greater but less than 3.7 must be installed and configured on your machine.
```

Comment:
Just to be sure, my understanding is that 3.7 doesn't work for developers (that need to push charts to registries) but it does work for end users. I'm not sure that this change is appropriate here.

Reply:
@roee88 the blueprint controller uses helm version deployed on the system, right? in that case the exact version of helm should be specified IIUC

Reply:
I think so too. the quick start contains only `helm install` commands which were not changed in helm 3.7.

Reply:
@sima I think that the helm version should be specified in `Module Helm Chart` in module file: ` Note that Helm 3.3 or above is required.`

Reply:
OK.  Will leave it as is here, and mention it in the contributing doc